examples/dtls-echo/Makefile

```diff
@@ -9,8 +9,8 @@ RIOTBASE ?= $(CURDIR)/../..
 
 # TinyDTLS only has support for 32-bit architectures ATM
 BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno chronos \
-                   msb-430 msb-430h telosb waspmote-pro wsn430-v1_3b wsn430-v1_4 \
-                   z1 jiminy-mega256rfr2 mega-xplained
+                   mips-malta msb-430 msb-430h pic32-clicker pic32-wifire \
+				   telosb waspmote-pro wsn430-v1_3b wsn430-v1_4 z1
```

Comment:
Looks like you introduced tabs here

Reply:
The jiminy-mega256rfr2 and mega-xplained have been removed and mips-malta, pics32* were added. Why are these changes required ?

Reply:
I run `make buildtest` without the BOARD_BLACKLIST parameter for trying to update the black list of boards. But, I just saw the big flaw in my logic here. I should have done it with a docker image or waited until Murdock complained. If it's OK, I'll do the former ASAP, otherwise I can revert the changes.